src/utils/validateIdentifierInputs.test.ts

```diff
@@ -189,4 +189,13 @@ describe("validateIdentifierInputs", () => {
       ])
     ).toThrow("Enter or scan a different code.");
   });
+
+  it("should throw the specific error if the identifierInput is payment receipt number", () => {
```

Comment:
maybe the test case can be named more specifically? Like in this case, `if the identifierInput value is empty`. 

Also, will other types of "STRING" like `spaces` or `emojis` be validated?

Reply:
yeap have added a new type, `PAYMENT_RECEIPT` that uses regex validation and added more tests!